zokrates_book/src/sha256example.md

```diff
@@ -52,8 +52,8 @@ grep '~out' witness
 which should lead to the following output:
 
 ```sh
-~out_0 263561599766550617289250058199814760685
 ~out_1 65303172752238645975888084098459749904
+~out_0 263561599766550617289250058199814760685
```

Comment:
This is the order I got when running the command.

Reply:
Interesting. We need to investigate the cause of this

Reply:
Do you get the original doc version if you run it? I can remove this change for now while we investigate.